source/app/blueprints/case/case_tasks_routes.py

```
@@ -0,0 +1,429 @@
+
+        return response_error("Unable to update task for internal reasons")
+
+    except marshmallow.exceptions.ValidationError as e:
+        return response_error(msg="Data error", data=e.messages)
+
+
+@case_tasks_blueprint.route('/case/tasks/delete/<int:cur_id>', methods=['POST'])
+@ac_api_case_requires(CaseAccessLevel.full_access)
+def case_delete_task(cur_id, caseid):
+    call_modules_hook('on_preload_task_delete', data=cur_id, caseid=caseid)
+    task = get_task_with_assignees(task_id=cur_id, case_id=caseid)
```

Ruff (0.8.2)

Undefined name `get_task_with_assignees`

(F821)